lib/error/try_test.cc

```diff
@@ -0,0 +1,17 @@
+#include "lib/error/try.h"
+#include "gtest/gtest.h"
+#include "lib/error/status.h"
+
+using namespace error;
+
+const Status kDefault(Code(1), "default");
+const Status kOther(Code(2), "other");
+
+TEST(TryTest, Status) {
```

Comment:
would probably try to add a couple test for a few contrived use cases of macros, to see the macro itself behaving correctly?

Eg, a TRY as single expression in an if followed by others? A tmp variable already declared? Not sure about the value, but those are cases where it is easy to introduce a macro error, and have it work in most cases but compile fail or lead to the wrong result in others?